UI/css/main.css

```diff
@@ -90,17 +92,21 @@ padding-bottom: 20px;
 }
 
 .banner{
-    background-image: url('../img/01.jpg');
+    background-image: url('../img/13.jpg');
     background-size: cover;
-    height: 350px;   
-    padding: 20px 0 20px 50px; 
 }
+.overlay{
+    background-color: rgba(0, 0, 0, 0.7);
+     padding: 50px;
 
+}
```

Comment:
ensure that there is a space after a closing curly brace and the next class/element/id style definition. Something like this:

```css
.overlay{
    background-color: rgba(0, 0, 0, 0.7);
     padding: 50px;
 }

.overlay{
    background-color: rgba(0, 0, 0, 0.7);
     padding: 50px;
 }
```

Reply:
Noted. Thanks